internal/btesting/btesting.go

```diff
@@ -0,0 +1,211 @@
+package btesting
+
+import (
+	"flag"
+	"fmt"
+	"github.com/stretchr/testify/require"
+	bolt "go.etcd.io/bbolt"
```

Comment:
Please put them in a separate group. This comment applies to almost all other files.